cypress/integration/smoke.spec.ts

```diff
@@ -34,30 +35,91 @@ function addCommonProvisioningADXDatasource(ADXProvisions: ADXProvision[]) {
     },
     expectedAlertMessage: 'Success',
   });
-}
+};
 
 e2e.scenario({
-  describeName: 'Add ADX datasource',
-  itName: 'fills out datasource connection configuration',
+  describeName: 'Add ADX datasource Import Dashoard',
+  itName: 'fills out datasource connection configuration and imports JSON dashboard',
   scenario: () => {
     e2e()
       .readProvisions(['datasources/adx.yaml'])
       .then((ADXProvisions: ADXProvision[]) => {
         addCommonProvisioningADXDatasource(ADXProvisions);
+        e2e.flows.importDashboard(TEST_DASHBOARD);
       });
   },
 });
 
 e2e.scenario({
-  describeName: 'Import dashboard',
-  itName: 'adds JSON',
+  describeName: 'Creates Panel run KQL query',
+  itName: 'fills out datasource connection configuration, adds panel and runs query',
   scenario: () => {
     e2e()
       .readProvisions(['datasources/adx.yaml'])
       .then((ADXProvisions: ADXProvision[]) => {
         addCommonProvisioningADXDatasource(ADXProvisions);
 
-        e2e.flows.importDashboard(TEST_DASHBOARD);
+        e2e.flows.addDashboard({
+          timeRange: {
+            from: '2019-10-05 19:00:00',
+            to: '2019-10-10 19:00:00',
+          },
+          variables: [],
+        });
+
+        e2e.flows.addPanel({
+          matchScreenshot: false,
+          visitDashboardAtStart: false,
+          queriesForm: () => {
+            e2eSelectors.queryEditor.editKQL.button().click({ force: true });
+            e2eSelectors.queryEditor.database.input().click({ force: true });
+            cy.contains('PerfTest').click();
+            e2eSelectors.queryEditor.codeEditorLegacy
+              .textarea()
+              .should('be.visible')
+              .clear()
+              .click()
+              .type(`PerfTest | where $__timeFilter(_Timestamp_) | order by _Timestamp_ asc`);
+            e2eSelectors.queryEditor.runQuery.button().click({ force: true });
+            cy.get('.panel-loading');
+            cy.get('.panel-loading', { timeout: 10000 }).should('not.exist');
+          },
+        });
+      });
```

Comment:
Tried this and this scenario was the one failing for me. In particular `clear` was not clearing the textarea (probably something weird in our component). Also, `.panel-loading' was not being rendered either. I modified those two and this seems to work for me:

```suggestion
        e2e.flows.addPanel({
          matchScreenshot: false,
          visitDashboardAtStart: false,
          queriesForm: () => {
            e2eSelectors.queryEditor.database.input().click({ force: true });
            cy.contains('PerfTest').click();
            e2eSelectors.queryEditor.editKQL.button().click({ force: true });
            e2eSelectors.queryEditor.codeEditorLegacy
              .textarea()
              .should('be.visible')
              .click({ force: true })
              .type('{del}'.repeat(100))
              .type(`PerfTest | where $__timeFilter(_Timestamp_) | order by _Timestamp_ asc`);
            e2eSelectors.queryEditor.runQuery.button().click({ force: true });
            cy.contains('Lonely period range deg');
          },
        });
      });
```

hope that helps!